pynsee/localdata/get_geo_list.py

```diff
@@ -259,12 +228,13 @@ def get_geo_list(geo=None, date=None, update=False):
     else:
         try:
             df_geo = pd.read_pickle(file_localdata)
-        except:
+        except Exception:

```

Comment:
I don't think this makes any difference, does it?

Reply:
This is considered bad practice in accordance with [pep 8](https://peps.python.org/pep-0008/#programming-recommendations) :
`A bare except: clause will catch SystemExit and KeyboardInterrupt exceptions, making it harder to interrupt a program with Control-C, and can disguise other problems. If you want to catch all exceptions that signal program errors, use except Exception: (bare except is equivalent to except BaseException:).`